ci/infra/testrunner/skuba/skuba.py

```diff
@@ -185,6 +185,15 @@ def _run_skuba(self, cmd, cwd=None):
         if cwd is None:
            cwd=self.cwd
 
+        if verbosity is None:
+           verbosity = self.conf.skuba.verbosity  # noqa: E111
+        try:
+            v = int(verbosity)
+        except ValueError:
+            v = 2  # default verbosity if unspecified / error
+            logger.warn(f"verbosity '{verbosity}' is not an int; using {v}")
```

Comment:
I would not swallow the exception silently. I would rise using the message you are using for the log.

Reply:
I guess the question is if this configuration error should halt execution.  Initially, I thought "no", but maybe you're right; maybe it should die if an invalid value is provided.

Reply:
I switched it to log as an error and raise a generic exception.  Should it just `sys.exit` instead, as there are a couple of places where _run_skuba is called inside a try block that (mildly annoyingly) catches all exceptions?  I feel like raising `Exception` is probably the technically better route, but am open to suggestions.

Reply:
I generally against exiting from inside a library. Exception should be a prefered way. And logging and raising an exception is generally considered an anti-pattern (we have been removing this from testrunner)

Reply:
I agree regarding log + exception (and regarding exit). However, several calls to this `catch exception` without emitting the exception text, so I did this out of concern about the cause of the error being lost.  Little is more annoying as a user than an error which doesn't say what's wrong. :)

Would it be better to leave this in and remove it in a separate PR improving the relevant exception handling, or just go ahead and improve the `catch` blocks' text in this PR?

Reply:
Switched to raise customized ValueError exception.